audio/flac2mp3/Portfile

```diff
@@ -30,11 +29,10 @@ checksums           rmd160  eb5d32d163f217ccbd5c2c969159486adcf922be \
 distname            ${name}
 
 # add perl variants
-# set default_branch to 5.28
 perl5.require_variant   yes
 perl5.conflict_variants yes
-perl5.branches          5.26 5.28 5.30
-perl5.default_branch    5.28
+perl5.branches          5.28 5.30 5.32 5.34
```

Comment:
Honestly I don't see much point in adding variants for 5.32 etc., but it doesn't really hurt either.